test/spec/geo/leaflet/leaflet-tiled-layer-view.spec.js

```diff
@@ -2,70 +2,132 @@ var Backbone = require('backbone');
+          maxZoom: 12,
+          minZoom: 1,
+          tms: true
+        });
+      });
+
+      it('should update several attributes', function () {
+        expect(this.layerView.leafletLayer.options.subdomains).toEqual(['b']);
+        expect(this.layerView.leafletLayer.options.attribution).toBe('carto');
+        expect(this.layerView.leafletLayer.options.errorTileUrl).toBe('http://matall.in');
+        expect(this.layerView.leafletLayer.options.maxZoom).toBe(12);
+        expect(this.layerView.leafletLayer.options.minZoom).toBe(1);
+        expect(this.layerView.leafletLayer.options.opacity).toBe(1);
+        expect(this.layerView.leafletLayer.options.tms).toBeTruthy();
+      });
+
+      it('should update url', function () {
+        expect(this.layerView.leafletLayer._url).toBe('http://hello.com');
+      });
+    });
+
+    it('should trigger load and loading events', function () {
+      var loadCallback = jasmine.createSpy('loadCallback');
+      this.layerView.bind('load', loadCallback);
+      this.layerView.leafletLayer.fire('load');
+      expect(loadCallback).toHaveBeenCalled();
+
+      var loadingCallback = jasmine.createSpy('loadingCallback');
+      this.layerView.bind('loading', loadingCallback);
+      this.layerView.leafletLayer.fire('loading');
+      expect(loadingCallback).toHaveBeenCalled();
+    });
   });
 
-  describe('._modelUpdated', function () {
+  describe('supports high resolution screens', function () {
     beforeEach(function () {
-      this.layerModel.set({
-        subdomains: ['b'],
-        urlTemplate: 'http://hello.com',
-        attribution: 'carto',
-        maxZoom: 12,
-        minZoom: 1,
-        tms: true
+      this.layerModel = new Backbone.Model({
+        urlTemplate: 'http://whatever.com',
+        urlTemplate2x: 'http://whatever.com2x',
+        subdomains: 'a',
+        opacity: 1,
+        attribution: 'CARTO',
+        errorTileUrl: 'http://matall.in',
+        maxZoom: 10,
+        minZoom: 0,
+        tms: false
+      });
+      this.leafletMap = jasmine.createSpy('leafletMap');
+      this.layerView = new LeafletTiledLayerView(this.layerModel, {
+        nativeMap: this.leafletMap
       });
     });
 
-    it('should update several attributes', function () {
-      expect(this.layerView.leafletLayer.options.subdomains).toEqual(['b']);
-      expect(this.layerView.leafletLayer.options.attribution).toBe('carto');
-      expect(this.layerView.leafletLayer.options.errorTileUrl).toBe('http://matall.in');
-      expect(this.layerView.leafletLayer.options.maxZoom).toBe(12);
-      expect(this.layerView.leafletLayer.options.minZoom).toBe(1);
-      expect(this.layerView.leafletLayer.options.opacity).toBe(1);
-      expect(this.layerView.leafletLayer.options.tms).toBeTruthy();
-    });
+    describe('is high resolution screen', function () {
+      describe('._modelUpdated', function () {
+        it('to a model without high resolution enabled', function () {
```

Comment:
[same here](https://github.com/CartoDB/cartodb.js/pull/2074/files#r176676515).

So I´d try to transform this:
> leaflet-tiled-layer-view: supports high resolution screens is high resolution screen ._modelUpdated to a model 
without high resolution enabled

Into

> leaflet-tiled-layer-view: when the layer-model supports high-resolution screens and the screen is high resolution ._modelUpdated should pick the high-resolution option as layer URL